src/Redux/api.tsx

```diff
@@ -634,19 +644,23 @@ const routes = {
   // States
   statesList: {
     path: "/api/v1/state/",
+    TRes: Type<IStateListResponse>(),
   },
 
   getState: {
     path: "/api/v1/state/{id}/",
+    TRes: Type<StateModel>(),
   },
 
   // Districts
 
   getDistrict: {
     path: "/api/v1/district/{id}/",
+    TRes: Type<DistrictModel>(),
   },
   getDistrictByState: {
     path: "/api/v1/state/{id}/districts/",
+    TRes: Type<StateModel[]>(),
```

Comment:
Why is districts API returning sates list?

```suggestion
    TRes: Type<DistrictModel[]>(),
```